Tests/OptimizelyTests-Common/DecisionServiceTests_Features.swift

```diff
@@ -253,10 +301,58 @@ extension DecisionServiceTests_Features {
         XCTAssertNil(variation)
     }
     
-    func testGetVariationForFeatureRolloutMultiple() {
-        // add tests for last rollout handling
+    func testGetVariationForFeatureRolloutFallbackRule() {
+        self.config.project.rollouts = [try! OTUtils.model(from: sampleRolloutData)]
+        self.config.project.rollouts[0].experiments[0].trafficAllocation[0].endOfRange = 0
+        featureFlag.rolloutId = kRolloutId
+        self.config.project.featureFlags = [featureFlag]
+        
+        let variation = self.decisionService.getVariationForFeatureRollout(config: config,
+                                                                           featureFlag: featureFlag,
+                                                                           userId: kUserId,
+                                                                           attributes: kAttributesAgeMatch)
+        XCTAssert(variation!.key == kVariationKeyC)
     }
     
+    func testGetVariationForFeatureRolloutEvaluatesNextIfAudienceEvaluationFails() {
+        self.config.project.rollouts = [try! OTUtils.model(from: sampleRolloutData)]
+        featureFlag.rolloutId = kRolloutId
+        self.config.project.featureFlags = [featureFlag]
+        
+        let variation = self.decisionService.getVariationForFeatureRollout(config: config,
+                                                                           featureFlag: featureFlag,
+                                                                           userId: kUserId,
+                                                                           attributes: kAttributesCountryMatch)
```

Comment:
I think, we need different audience for 2nd rollout rule. CountryMatch is used for feature-experiment. So no guarantee if it's passed down to rollout.